src/Simulation/EntryPointDriver/Azure.cs

```diff
@@ -0,0 +1,181 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.CommandLine.Parsing;
+using System.Threading.Tasks;
+using Microsoft.Quantum.Runtime;
+
+namespace Microsoft.Quantum.CsharpGeneration.EntryPointDriver
+{
+    /// <summary>
+    /// Provides entry point submission to Azure Quantum.
+    /// </summary>
+    internal static class Azure
+    {
+        /// <summary>
+        /// Submits the entry point to Azure Quantum.
+        /// </summary>
+        /// <param name="entryPoint">The entry point.</param>
+        /// <param name="parseResult">The command-line parsing result.</param>
+        /// <param name="settings">The submission settings.</param>
+        /// <typeparam name="TIn">The entry point's argument type.</typeparam>
+        /// <typeparam name="TOut">The entry point's return type.</typeparam>
+        internal static async Task<int> Submit<TIn, TOut>(
+            IEntryPoint<TIn, TOut> entryPoint, ParseResult parseResult, AzureSettings settings)
+        {
+            var machine = CreateMachine(settings);
+            if (machine is null)
+            {
+                DisplayUnknownTargetError(settings.Target);
+                return 1;
+            }
+
+            var job = await machine.SubmitAsync(entryPoint.Info, entryPoint.CreateArgument(parseResult));
+            switch (settings.Output)
+            {
+                case OutputFormat.FriendlyUri:
+                    Console.WriteLine("Job submitted. To track your job status and see the results use:");
+                    Console.WriteLine();
+                    // TODO: Show the friendly URI.
```

Comment:
What's this is a todo rather than done?

Reply:
There is not a way to get the friendly URI from the job at the moment. I think @cesarzc might have something planned for that?

Reply:
Ah, I see. Just curious.